README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do List 
+
+> This is an interactive tool to update and remind yourself of what you need to do.
+
+![screenshot](./img/screen_shot.png)
+
+## Built With
+
+- JavaScript
+- Webpack
+- HTML & CSS
+
+## Live Demo
+
+[Live Demo Link](https://priceless-goldstine-d252ba.netlify.app/)
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+Go to [My Repo](https://github.com/sneeu-leeu/To-Do-List/tree/development) and clone the project by opening your terminal and entering "git clone" + "The link supplied".
+
+cd into the repo
+
```

Comment:
- [ ] Let's include the `npm install` command to further guide users to install node packages which happens before `npm start`